scrapers/TheScoreGroup.yml

```diff
@@ -30,20 +33,44 @@ xPathScrapers:
               - regex: ..,
                 with:
           - parseDate: January 2 2006
-      Details:
-        selector: //div[@class="row"]/div/div[@class="p-desc"]/text()
-        concat: "\n"
+      Details: &details
+        selector: //div[@class="p-desc p-3"]//text()
+        postProcess:
+          - replace:
+              - regex: Read More »
+                with:
+        concat: "\n\n"
```

Comment:
This leads to some funny extra paragraphs for any descriptions that use HTML tags for things like italicized text, see [this XL Girls scene](https://www.xlgirls.com/bbw-videos/Juliana-Rodriguez/72936/) for instance

It's better than the old scraper, though, because that didn't return anything at all for this scene

Reply:
Yes, I caught that too and could not think of a way to address it. That issue is a tricky one.

Reply:
Figured out a way to fix the "extra paragraphs". All other issues addressed. Scraper should be ready to review.